test/kb_Metrics_server_test.py

```diff
@@ -118,6 +125,175 @@ def getImpl(self):
     def getContext(self):
         return self.__class__.ctx
 
+    # Uncomment to skip this test
+    # @unittest.skip("skipped test_MetricsMongoDBs_constructor")
+    def test_MetricsMongoDBs_constructor(self):
+        # testing if the db is connected and handshakes cab be made
+        exec_cur = self.dbi.metricsDBs['exec_engine']['exec_tasks'].find()
+        self.assertEqual(len(list(exec_cur)), 84)
+        ws_cur = self.dbi.metricsDBs['workspace']['workspaces'].find()
+        self.assertEqual(len(list(ws_cur)), 27)
+        wsobj_cur = self.dbi.metricsDBs['workspace']['workspaceObjects'].find()
+        self.assertEqual(len(list(wsobj_cur)), 15)
+        ujs_cur = self.dbi.metricsDBs['userjobstate']['jobstate'].find()
+        self.assertEqual(len(list(ujs_cur)), 36)
+        act_cur = self.dbi.metricsDBs['metrics']['daily_activities'].find()
+        self.assertEqual(len(list(act_cur)), 1603)
+
+    # Uncomment to skip this test
+    # @unittest.skip("skipped test_MetricsMongoDBs_list_exec_tasks")
+    def test_MetricsMongoDBs_list_exec_tasks(self):
+        minTime = 1500000932952
+        maxTime = 1500046845591
+
+        # testing list_exec_tasks return data
+        exec_tasks = self.dbi.list_exec_tasks(minTime, maxTime)
+        self.assertEqual(len(exec_tasks), 3)
+
+    # Uncomment to skip this test
+    # @unittest.skip("skipped test_MetricsMongoDBs_list_user_objects_from_wsobjs")
+    def test_MetricsMongoDBs_list_user_objects_from_wsobjs(self):
+        minTime = 1519668635550
+        maxTime = 1519768865840
+
+        # testing list_user_objects_from_wsobjs return data
+        user_objs = self.dbi.list_user_objects_from_wsobjs(minTime, maxTime)
+        self.assertEqual(len(user_objs), 15)
+        self.assertIn('workspace_id', user_objs[0])
```

Comment:
same here. Testing what the key value actually is instead of just testing it's existence.

Reply:
Testing existence is the first step. I will add the k-v tests.